versions/development/SPEC.md

```diff
@@ -358,6 +358,96 @@ Strings can also contain the following types of escape sequences:
 * A hexadecimal escape code starts with `\x`, followed by 2 hexadecimal digits `0-9a-fA-F`. 
 * A unicode code point starts with `\u` followed by 4 hexadecimal characters or `\U` followed by 8 hexadecimal characters `0-9a-fA-F`.
 
+##### Multi-line Strings
+
+Strings that begin with `<<<` and end with `>>>` may span multiple lines.
+
+```wdl
+String s = <<<
+  This is a
+  multi-line string!
+>>>
+
+In multi-line strings, leading *whitespace* is removed according to the following rules. In the context of multi-line strings, whitespace refers to space (`\x20`) and tab characters only and is treated differently from newline characters.
+
+1. Remove all whitespace following the opening `<<<`, up to and including a newline (if any).
+2. Remove all whitespace preceeding the closing `>>>`, up to and including a newline (if any).
+3 Remove all line continuations and subsequent white space.
+  * A line continuation (`\`) comes at the end of a line (i.e. immediately preceding a newline) and indicates that two consecutive lines are actually the same line (e.g. when breaking a long line for better readability).
+  * Removing a line continuation means removing the `\` character, the immediately following newline, and all the whitespace preceeding the next non-whitespace character or end of line (whichever comes first).
+4. Use all remaining non-*blank* lines to determine the *common leading whitespace*.
+  * A blank line contains zero or more whitespace characters followed by a newline.
+  * Common leading whitespace is the minimum number of whitespace characters occuring before the first non-whitespace character in a non-blank line.
+  * Each whitespace character is counted once regardless of whether it is a space or tab (so care should be taken when mixing whitespace characters).
+5. Remove common leading whitespace from each line.
+
+```wdl
+# all of these strings evaluate to "hello world"
```

Comment:
comment should have two spaces between hello & world

Reply:
thanks, fixed